demos/projects/NXP/mimxrt1060/port/azure_iot_flash_platform.c

```diff
@@ -53,6 +53,15 @@ AzureIoTResult_t AzureIoTPlatform_Init( AzureADUImage_t * const pxAduImage )
     return eAzureIoTSuccess;
 }
 
+int64_t AzureIoTPlatform_GetSingleFlashBootBankSize()
+{
+    // TODO: Fill in returning of flash bank size (to make sure we don't install an image larger than that)
```

Comment:
if this is the final PR, why is there a todo?

Reply:
This file is a stub for the NXP sample - all of the other functions have TODO for implementation as well